src/engine/shared/entities/player.lua

```diff
@@ -8,6 +8,8 @@
 local players      = player and player.players      or {}
 local lastPlayerId = player and player.lastPlayerId or 0
 
+local nameTable = {"Andrew", "James", "Ryan", "Brian", "Adam"}

```

Comment:
Denied, we should be using some sort of userinfo convar for this, I've yet to implement it because I'm a dingus